kalite/distributed/static/js/distributed/distributed-server.js

```diff
@@ -425,3 +424,19 @@ $(function() {
     });
 
 });
+
+
+/*This function addresses Bootstrap's limitation of having a dropdown menu in an already collapsed menu*/
```

Comment:
I... don't think I'll have time during this cycle to finish this *shamefully uncomments*